python/py-tensorflow1/Portfile

```diff
@@ -48,21 +50,24 @@ patch.pre_args      -p1
 
 # Required java version
 java.version        11+
-# JDK port to install if required java not found
-java.fallback       openjdk12
+# LTS JDK port to install if required java not found
+java.fallback       openjdk11
 # JDK only needed at build time, but java PG sets lib dependency so
 # declare no conflict to allow redistribution of binaries.
-license_noconflict  openjdk12
+license_noconflict  ${java.fallback}
 
 # The oldest Xcode version to use default Xcode compiler
 # Note setting here should be in sync with that in bazel
 set tf_min_xcode    9.0
 
 patchfiles-append Fix-build-failures-for-python-3.8.patch
+
+# Need clock_gettime()
+legacysupport.newest_darwin_requires_legacy 15
+
+# Delete when updating from 1.15.3
 if {${os.major} < 16} {
-    # Work around for issues with clock_gettime(CLOCK_REALTIME, &ts);
-    # https://stackoverflow.com/questions/5167269/clock-gettime-alternative-in-mac-os-x
-    patchfiles-append CLOCK_REALTIME-Older-OSX.patch
+    incr revision 1
 }
```

Comment:
If `clock_gettime()` is indeed why this patch was needed, then I think it would be better to instead now use the `legacysupport` portgroup for that purpose (one less patch to maintain).

I am not certain that the implementation used in the patch is identical to the one provided by `legacysupport`, so I believe the revision should be incremented at least for affected OS versions. Rather than incrementing the revision for unaffected OS versions, I have added version incrementer (and a note to remove it when updating), unless I should instead use a separate revision, or just increment the revision for all OS versions.

Reply:
Does removing this patch actually work ? I recall that the tensorflow/bazel build system does not behalf like others, and I am not sure the legactysupport fixes fully work here ?